website/source/intro/getting-started/change.html.md

```diff
@@ -33,7 +33,7 @@ resource "aws_instance" "example" {
 }
 ```
 
-~> **Note:** EC2 Classic users please use AMI `ami-2106ed4c` and type `t1.micro`
+~> **Note:** EC2 Classic users please use AMI `ami-8ac04ff9` and type `t2.micro`
```

Comment:
We have to leave t1 instances in the README for EC2 Classic Users. T2 instances can only be launched into an EC2-VPC environment